deep_qa/models/multiple_choice_qa/tuple_inference.py

```diff
@@ -6,12 +6,11 @@
 import numpy
 
 from deep_qa.data.instances.tuple_inference_instance import TupleInferenceInstance
-from deep_qa.layers.tuple_matchers.word_overlap_tuple_matcher import WordOverlapTupleMatcher
-from deep_qa.layers.tuple_matchers import tuple_matchers
-from ...layers.attention.masked_softmax import MaskedSoftmax
-from ...layers.backend.repeat import Repeat
-from ...layers.noisy_or import NoisyOr
-from ...layers.wrappers.time_distributed_with_mask import TimeDistributedWithMask
+from deep_qa.layers.tuple_matchers import tuple_matchers, WordOverlapTupleMatcher
```

Comment:
This `deep_qa` prefix should just be `..`, like the others, along with the `Instance` line above.  Looks like I missed this on the initial PR.  Can you fix it?